packages/calypso-e2e/src/lib/flows/gutenboarding-flow.ts

```diff
@@ -200,4 +200,27 @@ export class GutenboardingFlow {
 		const elementHandle = await this.page.waitForSelector( `${ selectors.planItem }.has-badge` );
 		await elementHandle.waitForSelector( `div:text-is("${ name }")` );
 	}
+
+	/* Other actions */
+
+	/**
+	 * Clicks on the language picker on top right of the Gutenboarding flow.
+	 */
+	async clickLanguagePicker(): Promise< void > {
+		await Promise.all( [
+			this.page.waitForNavigation(),
+			this.page.click( 'a:has(.gutenboarding__header-site-language-label)' ),
```

Comment:
Curious as to what the distinction is between selectors that are included in the selectors object and those that aren't.

Reply:
Ah, this was just my mistake in leaving it here. Thanks for catching it!

In general, I tend to put the selector up in the object if one of the following holds true:
1. it is used at least twice or more in the file.
2. it is long and unwieldy.
3. it is used in modified (but not exact form) through the file.

An example for the last one might be say, a button with class `button:has-text("Continue")` that might be altered with the additional class `.is-selected`. In such case I tend to move the base part of the selector to the object and use it like `${selectors.continueButton}.is-selected`.

Reply:
Fixed in e042fa9f4aca36910a20e277a243e1f68026a26a.